src/dev/flang/ast/Type.java

```diff
@@ -1061,6 +1069,19 @@ public AbstractFeature featureOfType()
   }
 
 
+  /**
+   * Is this the type of a type feature, e.g., the type of `(list
+   * i32).type`. false is this is unknown since type was not resolved yet.
```

Comment:
I find this comment to be misleading since `false` can also mean that it is actually known to not be the type of a type feature.

Reply:
dto.